pkg/plugins/diff/service_test.go

```diff
@@ -48,3 +51,105 @@ func TestService_RunDiff(t *testing.T) {
 		})
 	}
 }
+
+func Test_registerPlugins(t *testing.T) {
```

Comment:
Great coverage! Consider adding a case where you have more than a single plugin to monitor the wanted behavior when you have partial success in registration.

Reply:
done